doc/userguide/performance/rule-profiling.rst

```diff
@@ -1,6 +1,14 @@
 Rule Profiling
 ==============
 
+Suricata can generate a rules performance report at the end of each session, if
+built with the ``enable-profiling`` option (see configuring :ref:`Rule profiling<rule-and-packet-profiling-settings>`).
+
+Rule profiling can also be enabled by building the engine with ``enable-profiling-rules``
+and using the unix socket to dumpt the report (see :ref:`rule-management-rules-profiling`).
```

Comment:
typo `dumpt`

Reply:
Looks good to me besides this typo

Reply:
The PR was more correct than imagined. It was indeed a "typo" update! :P